api-doc.md

```diff
@@ -298,3 +298,42 @@
             </url>
         </urlset>
 ```
+
+### 4. Send New Leave Request mail
+
+```
+- Method : POST
+- Endpoint : /api/leave/new
+- Description : API for sending mail to HR for New leave request
+- Request:
+    - Headers : none
+    - Body :
+        {
+            "name": "mansi dhameliya",
+            "date": "3 jan 2023",
+            "status": "request",
+            "receiver": "mansi@gmail.com",
+        }
+
+
+- Response :
+
+    - If API is not found,
+    - Status Code: 404 not found
+    - Data: no response data
+
+    - If any server error is there then,
+    - Status Code: 500 Internal server error
+    - Data: no response data
+
+    - If request will success ,
+    - Status Code: 200 Ok
+    - Headers : none
+    - Data :
+               {
+            "name": "mansi dhameliya",
```

Comment:
I think, no need for this response, we can send messages like, `Leave applied successfully` or `Leave request has been sent successfully`.